crates/lox-time/src/base_time.rs

```diff
@@ -73,6 +73,21 @@ impl BaseTime {
         Self::from_date_and_utc_timestamp(dt.date(), dt.time())
     }
 
+    /// Instantiates a [BaseTime] from a Julian Day Number.
+    pub fn from_julian_day_number(day_number: i32, epoch: Epoch) -> Self {
```

Comment:
How did you choose the order of parameters? My instinct would be to put the data the calculation is based on, `day_number` in this case, and flags or configuration first. But having the "value" first and the "unit" last like in math also makes sense.

We should choose a single convention, though.